### ISSUE TITLE
Add heading field to featured work block

### ISSUE THREAD

**Issue by haleyjohnso** [vigetlabs/npm#311]
## Overview

**Page or Element Name**
Featured work block

**Example URL**
https://live-national-public-media-v2.pantheonsite.io/sponsorship/podcast-sponsorship/

**Time estimate**
.5
	
## Details

**Summary:**
Please add the header field block option to the featured work block

**Current vs. Desired state:**
We'd like to be able to add a header above the featured work block instead of having it above the related work where they appear together.

**Screenshots**

![2020-01-17_17_35_24-Podcast_Sponsorship___National_Public_Media](https://user-images.githubusercontent.com/57419582/73892862-436e7380-4835-11ea-8712-17758e520cc4.png)

**Comment by ten1seven** [comment]
@haleyjohnso confirming that this is what they want the option to do (note header moved from middle to top of section:

![Screen Shot 2020-02-06 at 11 58 46 AM](https://user-images.githubusercontent.com/137020/73969163-22ab2a00-48d8-11ea-9c7f-8e6330e9e062.png)

**Comment by haleyjohnso** [comment]
Yes, confirmed. In sum, @ten1seven  you've added the ability for the featured work component to have a header without removing the ability for the related work to have a header. Was there anything about this ticket that gave you pause?

**Comment by ten1seven** [comment]
No, just confirming what the visual result should be.

**Comment by haleyjohnso** [comment]
I'm on staging (test) and while I **can** add a heading to the featured work block and have it appear as expected, it appears that the images in the Featured Work block and the Related Work block have disappeared.

<img width="1385" alt="Screen Shot 2020-02-11 at 12 46 47 PM" src="https://user-images.githubusercontent.com/57419582/74272955-cc215f80-4ccc-11ea-80f8-0104ba708a07.png">

**Comment by ten1seven** [comment]
I still see images on:
https://dev-national-public-media-v2.pantheonsite.io/sponsorship/podcast-sponsorship/
https://test-national-public-media-v2.pantheonsite.io/sponsorship/podcast-sponsorship/

Where are you testing it?

**Comment by haleyjohnso** [comment]
Weird. I think i've figured out what's happening. I'm on https://test-national-public-media-v2.pantheonsite.io/sponsorship/podcast-sponsorship/
BUT when I'm 'previewing' the page (which I normally do after making a change), the aren't any images. Just clearing the preview appendage (?preview_id=70&preview_nonce=58efcd3f5d&preview=true) from the URL makes the images come back. 

Ergo, disregard. There are no issues and the header works as expected.
